mis_builder_cash_flow/models/mis_cash_flow_forecast_line.py

```diff
@@ -0,0 +1,42 @@
+from odoo import fields, models, api, _
+from odoo.exceptions import ValidationError
+
+
+class MisCashFlowForecastLine(models.Model):
+
+    _name = 'mis.cash_flow.forecast_line'
+    _description = 'MIS Cash Flow Forecast Line'
+
+    date = fields.Date(
+        required=True,
+        index=True,
+    )
+    account_id = fields.Many2one(
+        comodel_name='account.account',
+        string='Account',
+        required=True,
+        help='The account of the forecast line is only for informative '
+        'purpose',
+    )
+    name = fields.Char(
+        required=True,
+        default='/',
+    )
+    balance = fields.Float(
+        required=True,
+    )
+    company_id = fields.Many2one(
+        'res.company',
+        string='Company',
+        required=True,
+        default=lambda self: self.env.user.company_id.id,
+        index=True,
+    )
+
+    @api.multi
+    @api.constrains('company_id', 'account_id')
+    def _check_company_id_employee_id(self):
```

Comment:
the name of the method could be more accurate
```suggestion
    def _check_company_id_account_id(self):
```